src/js/messaging/components/MessageDetails.jsx

```diff
@@ -76,35 +76,34 @@ class MessageDetails extends React.Component {
       </div>
     );
 
-    const inputId = `message-details-${messageId}`;
-    const compactInputId = `compact-${inputId}`;
-
     const compactSentDate = sentDate && (
       <span>{formattedDate(sentDate, { fromNow: true })}</span>
     );
 
     return (
       <div
-          ref="detailsControl"
           className="messaging-message-details-control"
           tabIndex="-1"
-          onBlur={this.hideDetails}
+          onBlur={() => {
+            if (this.state.expandedFull) {
```

Comment:
Hm, in this case, maybe it's better to just directly set the state than calling a method named `toggle*` if it's not really toggling...